MAUI/PDF-Viewer/Migration.md

```diff
@@ -432,9 +432,8 @@ To make migration from [Xamarin SfPdfViewer](https://www.syncfusion.com/xamarin-
 <div>{{'[TextMarkupSelectedEventArgs](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfPdfViewer.XForms.TextMarkupSelectedEventArgs.html)'| markdownify }} </div>
 </td>
 <td>{{'[AnnotationEventArgs ](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.AnnotationEventArgs.html)'| markdownify }}</td>
-<td></td>
+<td>Provides data for the AnnotationAdded, AnnotationSelected, AnnotationDeselected, AnnotationEdited, AnnotationRemoved events.</td>
```

Comment:
Please link APIs, as it a kind of API documents.

<img width="328" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/37c8ad33-83ce-42b3-9240-5ec7b0d9c9c5">

Reply:
Updated the feedback.